reference/src/client/include/sa_ta_types.h

```diff
@@ -82,14 +82,20 @@ typedef enum {
 } SA_COMMAND_ID;
 
 /**
- * The types of the TA parameters.
+ * The values of the TA parameters. The values correspond to the values in the GlobalPlatform TEE Client API
```

Comment:
Is the Global Platform TEE Client open source? Have we gotten the appropriate approvals to include? Do we need any attestations? How does this impact proprietary SecAPI implementations that don't employ the Global Platform implementation? Can we abstract this interface out or offer a customizable build?

Reply:
It did get flagged by blackduck and Martin approved it because it has the GlobalPlatform TEE reference in the comment. I don't think it matters for SecApi implementations that don't use Global Platform. They are just defines which are equivalent to the enums I had before.  If a vendor needs to change the values, they can put code into ta_client.c to map these values to what they need. With the old code, Realtek and Amlogic were doing the reverse, mapping my custom values to the Op-TEE values.